src/test/java/org/springframework/shell/SimpleShellCommandLineOptionsTest.java

```diff
@@ -0,0 +1,46 @@
+package org.springframework.shell;
+
+import java.io.IOException;
+import java.util.logging.Logger;
+
+import org.junit.Assert;
+
+import org.junit.Test;
+import org.springframework.shell.core.JLineShellComponent;
+import org.springframework.shell.support.logging.HandlerUtils;
+
+public class SimpleShellCommandLineOptionsTest {
+
+    @Test
+    public void testPorcelain() throws IOException {
+        try {
+            String[] args = {"--porcelain"};
+            Bootstrap bootstrap = new Bootstrap(args);
+            JLineShellComponent shell = bootstrap.getJLineShellComponent();
+
+            Assert.assertTrue("isPorcelain should be true", shell.getCommandLine().isPorcelain());
```

Comment:
Not sure but I don't believe so, since colorization can also be turned off if the terminal used doesn't support ANSI. So I can ensure colorization is turned off if the flag is present, but I can't ensure colorization is turned on if the flag isn't present.